sql/executor.go

```diff
@@ -206,8 +206,10 @@ func (e Executor) execStmt(stmt parser.Statement, params parameters, planMaker *
 						row.Values = append(row.Values, driver.Datum{
 							Payload: &driver.Datum_IntervalVal{IntervalVal: vt.Nanoseconds()},
 						})
+					case parser.DTuple:
+						return fmt.Errorf("unsupported result type: tuple, remove extra parenthesis around column %s", result.Columns[i])
```

Comment:
I dont think this advice is appropriate. other databases support the tuple return; it's fine that we do not, but this advice is inappropriate because it changes the semantics. The new verbiage in the default clause LGTM.